cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,32 @@
+class IframePage {
+
+    private w3schoolSiteUrl: string;
+    private defaulIframe: string;
+    private cssH1: string;
+    private tutorialSpan: string;
+
+    constructor() {
+        this.w3schoolSiteUrl = 'https://www.w3schools.com/html/html_iframe.asp';
+        this.defaulIframe = '[src="default.asp"]';
+        this.cssH1 = '#main > h1';
+        this.tutorialSpan = '#main > h1 > span';
+    }
+
+    public visit(): void {
+        cy.visit(this.w3schoolSiteUrl);
+    }
+
+    public getFrameTitle() {
+        return cy.iframe(this.defaulIframe);
+    }
+
+    public goToCssPageInFrame() {
+        cy.iframe(this.defaulIframe).find('a[title="CSS Tutorial"]').click();
```

Comment:
the text used in find should be a selector and should be a class property

Reply:
Done!